### DISCUSSION TITLE
Add more data to cities

### DISCUSSION THREAD

**Discussion by JEricFarias** [dr5hn/countries-states-cities-database#515] [Ideas]
Now cities come with id and name. Would be interest if we can add more data like the state name and region.

So we can group

**Comment by dr5hn** [comment]
@JEricFarias is Already on my todo list, will implement it soon.

**Reply by dreigningking** [reply]
I have a detailed list of all cities in Nigeria. Would you like me to share (SQL format of states & cities)

**Reply by dr5hn** [reply]
@dreigningking Yes, please that would be great. Thanks
